client/verta/verta/_cli/registry/create.py

```diff
@@ -36,7 +36,8 @@ def create_model(model_name, label, visibility, workspace):
 @click.option("--model", help="Path to the model.")
 @click.option("--artifact", type=(str, str), multiple=True, help="Path to the artifact required for the model. The format is --artifact artifact_key path_to_artifact.")
 @click.option("--workspace", "-w", help="Workspace to use.")
-def create_model_version(model_name, version_name, label, model, artifact, workspace):
+@click.option("--from-run", help="ID of the Experiment Run to enter into the model registry")
+def create_model_version(model_name, version_name, label, model, artifact, workspace, from_run):
```

Comment:
```bash
verta registry create registeredmodelversion MODEL_NAME VERSION_NAME --run-id="xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
```

Reply:
We should somehow highlight this option will ignore many others

Reply:
nit: registeredmodelversion is this common usage , no snake_case not camelCase? 
personal opinion aside , fine to do if this is common practice

Reply:
aws cli seems to use hyphenated words

Reply:
As an opposing example, kubectl doesn't separate words (`kubectl get persistentvolumeclaims`), though they do offer short names.

I have a loose preference for hyphenated nouns because I think they're more readable, though harder to type.

I'll start a thread about this